diff --git Makefile.am Makefile.am
index 76a834ef..c8f41e6e 100644
--- Makefile.am
+++ Makefile.am
@@ -50,6 +50,7 @@ libxml2_la_SOURCES = SAX.c entities.c encoding.c error.c parserInternals.c  \
 		valid.c xlink.c HTMLparser.c HTMLtree.c debugXML.c xpath.c  \
 		xpointer.c xinclude.c nanohttp.c nanoftp.c \
 		$(docb_sources) \
+		xmlvalidate.c \
 		catalog.c globals.c threads.c c14n.c xmlstring.c buf.c \
 		xmlregexp.c xmlschemas.c xmlschemastypes.c xmlunicode.c \
 		$(trio_sources) \
